src/input-mappings.js

```diff
@@ -1,47 +1,94 @@
-          dpadrightdown: "action_snap_rotate_right",
-          dpadcenterdown: "action_teleport_down", // @TODO once once #30 lands in aframe-teleport controls this just maps to "action_teleport_aim"
-          dpadcenterup: "action_teleport_up", // @TODO once once #30 lands in aframe-teleport controls this just maps to "action_teleport_teleport"
-          touchpadpressedaxismovex: "translateX",
-          touchpadpressedaxismovey: "translateZ",
-          touchpadbuttonup: "stop_moving"
-        },
-        "vive-controls": {
-          menudown: "action_mute"
-        },
-        "oculus-touch-controls": {
-          xbuttondown: "action_mute",
-          gripdown: "middle_ring_pinky_down",
-          gripup: "middle_ring_pinky_up",
-          thumbsticktouchstart: "thumb_down",
-          thumbsticktouchend: "thumb_up",
-          triggerdown: "index_down",
-          triggerup: "index_up"
+const inGameActions = {
+  // Define action sets here.
+  // An action set separates "driving" controls from "menu" controls.
+  // Only one action set is active at a time.
+  default: {
+    move: { label: "Move" },
+    snap_rotate_left: { label: "Snap Rotate Left" },
+    snap_rotate_right: { label: "Snap Rotate Right" },
+    action_mute: { label: "Mute" },
+    action_teleport_down: { label: "Teleport Aim" },
+    action_teleport_up: { label: "Teleport" },
+    action_share_screen: { label: "Share Screen" }
+  }
+};
+
+const config = {
+  behaviours: {
+    default: {
+      "oculus-touch-controls": {
+        joystick: "oculus_touch_joystick_dpad4"
+      },
+      "vive-controls": {
+        trackpad: "vive_trackpad_dpad4"
+      }
+    }
+  },
+  mappings: {
+    default: {
+      "vive-controls": {
+        menudown: "action_mute",
+        "trackpad.pressedmove": { left: "move" },
+        trackpad_dpad4_pressed_west_down: { right: "snap_rotate_left" },
+        trackpad_dpad4_pressed_east_down: { right: "snap_rotate_right" },
+        trackpad_dpad4_pressed_center_down: { right: "action_teleport_down" },
+        trackpadup: { right: "action_teleport_up" }
+      },
+      "oculus-touch-controls": {
+        joystick_dpad4_west: {
+          right: "snap_rotate_left"
         },
-        daydream: {
-          menudown: "action_mute"
+        joystick_dpad4_east: {
+          right: "snap_rotate_right"
         },
-        keyboard: {
-          m_press: "action_mute",
-          q_press: "action_snap_rotate_left",
-          e_press: "action_snap_rotate_right",
-          v_press: "action_share_screen",
-          w_down: "action_move_forward",
-          w_up: "action_dont_move_forward",
-          a_down: "action_move_left",
-          a_up: "action_dont_move_left",
-          s_down: "action_move_backward",
-          s_up: "action_dont_move_backward",
-          d_down: "action_move_right",
-          d_up: "action_dont_move_right"
-        }
+        xbuttondown: "action_mute",
+        gripdown: "middle_ring_pinky_down",
```

Comment:
I would either get rid of these grip actions (since they are not used) or add them to the inGameActions map above